File: src/Angular2MultiSPA/wwwroot/app/models/Order.ts
import { Component } from '@angular/core';

export class Order {
    id: number;
    customerId: number;
    orderDate: Date;
    shippedDate: Date;
    shipVia: number;
    freight: number;
    shipName: string;
    shipAddress: string;
    shipCity: string;
    shipRegion: string;
    shipPostalCode: string;
    shipCountry: string;
}

File: src/Angular2MultiSPA/ViewModels/Order.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Angular2MultiSPA.ViewModels
{
    public class Order
    {
        public int Id { get; set; }

        public string CustomerId { get; set; }

        public DateTime? OrderDate { get; set; }

        public DateTime? ShippedDate { get; set; }

        public int? ShipVia { get; set; }

        [DataType(DataType.Currency)]
        public decimal? Freight { get; set; }

        public string ShipName { get; set; }

        public string ShipAddress { get; set; }

        public string ShipCity { get; set; }

        public string ShipRegion { get; set; }

        [DataType(DataType.PostalCode)]
        public string ShipPostalCode { get; set; }

        public string ShipCountry { get; set; }
    }
}
